doc/modules/kernel_approximation.rst

```diff
@@ -4,11 +4,26 @@ Kernel Approximation
 ====================
 
 This submodule contains functions that approximate the feature mappings that
-correspond to certain kernels, as they are used for example in support vector
-machines (see :ref:`svm`).
-The following feature functions perform non-linear transformations of the
-input, which can serve as a basis for linear classification or other
-algorithms.
+correspond to certain kernels, as they are used, for example, by
```

Comment:
This section on kernel approximation shouldn't really be discussing ridge vs svm, which tend to use exact kernels.

Reply:
Solved in ff40a1dc